app/code/Magento/Developer/Console/Command/DiInfoCommand.php

```diff
@@ -173,4 +195,41 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         return \Magento\Framework\Console\Cli::RETURN_SUCCESS;
     }
+
+    /**
+     * Set Area for DI Configuration
+     *
+     * @param string $area
+     * @return void
+     * @throws \InvalidArgumentException
+     */
+    private function setDiArea($area): void
+    {
+        if ($this->validateAreaCodeFromInput($area)) {
+            $objectManager = ObjectManager::getInstance();
+
+            $areaOmConfiguration = $objectManager
```

Comment:
Consider injecting the ObjectManager and ConfigLoader dependencies via the constructor to avoid direct usage of the ObjectManager.

Reply:
I added ObjectManager to the constructor, but I don't quite understand about ConfigLoader ( its instance is not used, I just need the class name )

Reply:
Also, in case we add OM to the constructor in the current approach, we get a cyclic dependency, right?